french/dormir/reve-etrange.md

```diff
@@ -1 +1,11 @@
-Vous parvenez à vous rendormir mais faites le plus étrange des rêves, dansant le tango avec un éléphant rose !
\ No newline at end of file
+Vous parvenez à vous rendormir mais faites le plus étrange des rêves, dansant le tango avec un éléphant rose !
+
+Vous vous reveillez ce qui semble etre quelques heures plus tard, de tres bonne humeur mais les pieds meurtris pour une etrange raison.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.